packages/grid/test/keyboard-navigation-cell-button.test.js

```diff
@@ -0,0 +1,75 @@
+import { expect } from '@esm-bundle/chai';
+import { arrowLeft, arrowRight, aTimeout, fixtureSync } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import '../vaadin-grid.js';
+import { flushGrid } from './helpers.js';
+
+let grid;
+
+function getRowCell(rowIndex, cellIndex) {
+  return grid.$.items.children[rowIndex].children[cellIndex];
+}
+
+function getRowFirstCell(rowIndex) {
+  return getRowCell(rowIndex, 0);
+}
+
+describe('keyboard navigation - cell button', () => {
+  let cell;
+
+  beforeEach(async () => {
+    grid = fixtureSync(`
+      <vaadin-grid>
+        <vaadin-grid-column path="name" _cell-button></vaadin-grid-column>
```

Comment:
`_cell-button`: not familiar with that syntax, is it something Polymer specific? If so, should we rather set the property explicitly?

Reply:
Oh, I see that it just sets the attribute, which then sets the property. The underscore at the start threw me off here. Nevermind then.

Reply:
Use, it's how Polymer works. Changed to set a property instead.